File: python3/1013.py
def maiorAB(a, b):
    return int((a + b + abs(a - b))/2)

entrada = input().split()
a, b, c = entrada
a = maiorAB(int(a), int(b))
a = maiorAB(a, int(c))
print(a, "eh o maior")

File: solutions/uri/1 - Iniciante/1013/1013.rb
def maior(a, b)
	(a + b + (a-b).abs)/2
end

entrada = gets.chomp.split(" ")
a = entrada[0].to_i
b = entrada[1].to_i
c = entrada[2].to_i
a = maior(a, b)
a = maior(a, c)

puts (a.to_s + " eh o maior")